src/components/about/content-provenance-ar.jsx

```diff
@@ -20,7 +16,7 @@ function Content() {
         <h3>المصدر: جامعة بنسلفانيا (Penn)</h3>
         <p className="text">
           تعود مجموعة مكتبات جامعة بنسلفانيا المتكونة من أكثر من 600 قطعة مخطوطة قروسطية لجنيزا القاهرة إلى مصدرين. يتأتى الجزء الأكبر منها من مكتية جامعة دروبسي (Dropsie College) وهي أول مؤسسة معتمدة في العالم لإسناد شهائد الدكتوراه في العلوم اليهودية. وفقا للسجلات الداخلية للجامعة والقائمة الوصفية لقطع الجنيزا في فيلادلفيا (1924)، تتأتى حيازات دروبسي (Dropsie) من هدايا مقدمة من سايرس أدلر (Cyrus Adler) ودافيد و. أمرام (David W. Amram) وهربرت فريدنفالد (Herbert Friedenwald) وإفرايم لدرر (Ephraim Lederer) وماير سلزبرغر (Mayer Sulzberger) ومن شراء من طرف أرملة كامدن م. كوبرن (Camden M. Cobern) الذي توفي في سنة 1920. صنف هالبر (Halper) القطع ضمن سبعة مجالات  موضوعية هي الكتاب المقدس والمدراش والتلمود والليتورجيا والشعر العلماني والوثائق والرسائل والفلسفة والكابالا ومواد أخرى متنوعة. في سنة 1986 حول دروبسي إلى معهد لبحوث ما بعد الدكتوراه سمي معهد أننبرغ للبحوث (Annenberg Research Institute / ARI) وتم دمجه فيما بعد في جامعة بنسلفانيا سنة 1993. في سبتمبر من سنة 1986 وبعد فترة وجيزة من  إنشاء المعهد تم التبرع بمجموعة صغيرة إضافية لمعهد أننبرغ للبحوث من طرف جاك لونزر (Jack Lunzer) بالنيابة عن مجلس صندوق فالمادونا (Valmadonna). في سنة 1994 وضعت جنيزا دروبسي تحت إدارة مكتبات بنسلفانيا لتنضاف إلى المجموعة الموجودة من جنيزا القاهرة والمحفوظة في مكتبة الكتب النادرة والمخطوطات لمكتبات جامعة بنسلفانيا المعروفة اليوم باسم مركز كسلاك (Kislak) للمجموعات الخاصة والكتب النادرة والمخطوطات.
-          <a href="http://openn.library.upenn.edu/html/genizah_contents.html" target="_blank">http://openn.library.upenn.edu/html/genizah_contents.html</a></p>
+          <a href="http://openn.library.upenn.edu/html/genizah_contents.html" rel="noopener noreferrer" target="_blank">http://openn.library.upenn.edu/html/genizah_contents.html</a></p>
```

Comment:
I'm just reading up on the importance of `rel="noopener"` and `rel="noreferrer"`; I was hitherto unaware of the significant performance impact and the security issue. TIL.

Reply:
Now that I see the problem, I also see that there is a _small truck load_ of `<a target="_blank">` links that still don't have `rel="noopener noreferrer"` as a safety. This is another item on the future ToDo.